LOAN/LoanTypes/SecuredLoans.rdf

```diff
@@ -71,14 +71,14 @@
 		<rdfs:subClassOf rdf:resource="&fibo-loan-typ-sec;SecuredLoan"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-typ-sec;howSecured.2"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-typ-sec;SecuredByGuaranty"/>
+				<owl:onProperty rdf:resource="&fibo-fbc-dae-gty;hasGuarantor"/>
+				<owl:someValuesFrom rdf:resource="&fibo-fbc-dae-gty;Guarantor"/>
 			</owl:Restriction>
 		</rdfs:subClassOf>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-loan-loan-gty;guaranteedBy"/>
-				<owl:someValuesFrom rdf:resource="&fibo-loan-loan-gty;LoanGuarantor"/>
+				<owl:onProperty rdf:resource="&fibo-loan-typ-sec;howSecured.2"/>
```

Comment:
Maybe we should have an issue/pass to eliminate any remaining ".n" properties

Reply:
Hoping to do that when secured and unsecured loans are merged into loan core - these seem like fundamental concepts and are partially addressed there as well.   Note that there are still mortgage-specific concepts in LoanCore that need to be teased out / moved to mortgage loans as well.  LoanCore should only include the true core details.